app/views/recipes/edit.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1 class="flex justify-center mt-10 text-4xl font-bold decoration-sky-500 mb-5">Editing: <%= @recipe.name %></h1>
+<div class="flex justify-around my-9">
+  <div class="ml-9">
+    <%= link_to @recipe do %>
+      <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6">
+        <path stroke-linecap="round" stroke-linejoin="round" d="M10.5 19.5L3 12m0 0l7.5-7.5M3 12h18" />
+      </svg>
+    <% end %>
+  </div>
+  <div class="mr-10">
+    <%= link_to recipes_path do %>
+      <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6">
+        <path stroke-linecap="round" stroke-linejoin="round" d="M15.75 9V5.25A2.25 2.25 0 0013.5 3h-6a2.25 2.25 0 00-2.25 2.25v13.5A2.25 2.25 0 007.5 21h6a2.25 2.25 0 002.25-2.25V15m3 0l3-3m0 0l-3-3m3 3H9" />
```

Comment:
- The requirements say: Clicking on a recipe item should lead to recipe details, Page has to match the structure of the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/public_recipe_index.png?raw=true). Can you fix it?

![2](https://github.com/fezzopro/recipe_app/assets/63972670/2a32fda9-9b89-4aa9-b04a-f319efa57352)

Reply:
This change has been done well

Reply:
thanks bro.